operator/pkg/ciliumendpointslice/namespace.go

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ciliumendpointslice
+
+import (
+	"context"
+
+	k8s "github.com/cilium/cilium/pkg/annotation"
+	"github.com/cilium/cilium/pkg/k8s/resource"
+	slimcorev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+const (
+	priorityNamespaceAnnotation = "cilium.io/ces-namespace"
+)
+
+func (c *Controller) processNamespaceEvents(ctx context.Context) error {
+
```

Comment:
nit: Remove the empty line at the beginning.

Same below for `onNamespaceUpstert()` and `onNamespaceDelete()` (also last line is empty here).